modules/mysql/main.tf

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  default_tier                            = "db-n1-standard-1"
```

Comment:
Defaults should be encoded in variables instead of locals (if possible). Is there a reason these can't be?

Reply:
Sorry I can't remember that because it has passed a little while since I sent a PR.
Probably your review seems good. Moved: d86bddbf9ff3c732d16897e2c596ea5572f47910 and 5f102ec06513661a0030203a61e72ff534f6a0c4